engine/CSVParser.cc

```diff
@@ -342,15 +342,16 @@ namespace minsky
     string buf;
     typedef vector<string> Key;
     map<Key,double> tmpData;
+    multimap<Key,double> tmpAll; 
```

Comment:
multiMap still here

Reply:
I've removed it and will shortly commit the latest fixes you have suggested.